traefik/dynamic_conf/chain-jitsi-auth.yml

```
@@ -0,0 +1,22 @@
+http:
+  middlewares:
+    middlewares-secure-headers-jitsi:
+      headers:
+        accessControlAllowMethods:
+          - GET
+          - OPTIONS
+          - PUT
+        accessControlMaxAge: 100
+        hostsProxyHeaders:
+          - "X-Forwarded-Host"
+        stsSeconds: 63072000
+        stsIncludeSubdomains: true
+        stsPreload: true
+        forceSTSHeader: true
+        customFrameOptionsValue: "allow-from https:YOUR_URL"
```

Defining the middleware, using docker labels, will allow you to define this option dynamically.

```suggestion
        customFrameOptionsValue: "allow-from https:meet.${SITE:-localhost}"
```